alas_but_one.py

```diff
@@ -1,39 +1,108 @@
 import os
 import re
-from collections import Counter
+from spellchecker import SpellChecker
```

Comment:
Suggestion:
When adding a dependency that's not part of the python core library, can you make sure there's a corresponding requirements.txt file and entry so it's easy to set up with ``pip install -r requirements.txt``?